.travis.yml

```diff
@@ -1,5 +1,34 @@
 sudo: required
 dist: trusty
+
+env:
+  COMPOSE_VERSION: 1.24.1
+  DSPACE_REST_HOST: localhost
+  DSPACE_REST_PORT: 8080
+  DSPACE_REST_NAMESPACE: '/server/api'
+  DSPACE_REST_SSL: false
+
+before_install:
+  # Docker Compose Install
+  - curl -L https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - git clone https://github.com/DSpace-Labs/DSpace-Docker-Images.git
+
+install:
+  - docker-compose version
+  - docker-compose -f DSpace-Docker-Images/docker-compose-files/dspace-compose/d7.travis.yml up -d
+  - travis_retry yarn install
+  # allow starup time
+  #- sleep 60
+
+before_script:
+  # Startup the app
+  #- curl http://localhost:8080/
```

Comment:
Do we need to keep the various commented out lines in this script?  I'm not against us having the `curl` command here (uncommented) if it is quick & proves that we have Docker running.  But, if we don't need it really, we can likely just remove it (and the commented out `sleep` stuff above)